daprdocs/content/en/reference/components-reference/supported-pubsub/setup-jetstream.md

```diff
@@ -146,6 +143,31 @@ It is essential to create a NATS JetStream for a specific subject. For example,
 nats -s localhost:4222 stream add myStream --subjects mySubject
 ```
 
+## Example: Competing consumers pattern
+
+Let's say you'd like each message to be processed by only one application or pod with the same app-id. Typically, the `consumerID` metadata spec helps you define competing consumers. However, `consumerID` is not supported in NATS JetStream.
```

Comment:
This does not explain what in the example below makes this work as `competing consumers` pattern.

I would point out that in NATS JetStream, you need to specify `durableName` and `queueGroupName`, so readers understand which config fields achieve this pattern.